api/tests/unit/features/test_unit_features_views.py

```diff
@@ -55,7 +55,7 @@ def test_list_feature_states_from_simple_view_set(
     )
 
     # When
-    with django_assert_num_queries(8):
+    with django_assert_num_queries(9):
```

Comment:
This is due to the addition of the `get_object_or_404` function call here: https://github.com/Flagsmith/flagsmith/pull/2382/files#diff-9e42c40eece4b1da492acaad6b58480687eab073351781b6a0fec0a91435df9aR543

This was necessary to simplify the code in the `versioning` module.